platform/vs/docker-physyncd-vs/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+PLATFORM_DIR=/usr/share/sonic/platform
+HWSKU_DIR=/usr/share/sonic/hwsku
+
+
+# Remove stale files if they exist
+rm -f /var/run/rsyslogd.pid
+
+supervisorctl start rsyslogd
+
+mkdir -p /etc/sai.d/
+
+# Create/Copy the sai.profile to /etc/sai.d/sai.profile
```

Comment:
change the comment : s/sai.profile/pai.profile/g

Reply:
thank you for the review comments, committed changes to address.